test/functional/wallet_basic.py

```diff
@@ -504,15 +504,58 @@ def run_test(self):
                                 {"address": address_to_import},
                                 {"spendable": False})
 
-            # 5. Import private key of the previously imported address on node1
+            # 5. Remove the address from node1
```

Comment:
nit: I don't think we need the numbering (makes it annoying to add more cases later)

Reply:
Removed (this was very annoying)